libraries/ui-library/package.json

```diff
@@ -45,18 +45,17 @@
     "@stencil/react-output-target": "^0.5.0",
     "@stencil/sass": "^3.0.1",
     "@stencil/vue-output-target": "^0.8.2",
-    "@types/jest": "^26.0.15",
+    "@types/jest": "27.0.3",
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.28.0",
-    "jest": "^26.6.3",
-    "jest-cli": "^26.6.3",
+    "jest": "27.4.5",
+    "jest-cli": "27.4.5",
     "material-icons": "^1.12.0",
-    "puppeteer": "^13.3.2",
-    "puppeteer-core": "^13.3.2",
+    "puppeteer": "^20.2.1",
```

Comment:
Can't we remove puppeteer completely, now that all e2e-tests are deleted?

Reply:
No we can't. I tried and if you check one of the last runs these fail then since the stencil-test package somehow underneath references puppeteer. It's annoying but we have to keep it for now due to that.